libcontainer/intelrdt/intelrdt.go

```diff
@@ -443,6 +443,11 @@ func (m *IntelRdtManager) GetPath() string {
 
 // Returns statistics for Intel RDT
 func (m *IntelRdtManager) GetStats() (*Stats, error) {
+	// If intelRdt is not specified in config
+	if m.Config.IntelRdt == nil {
+		return nil, nil
```

Comment:
Does the calling code handle a `nil, nil` return or will it panic?

Reply:
@crosbymichael 
> Does the calling code handle a nil, nil return or will it panic?

It will not panic. In caller linuxContainer.Stats(): libcontainer.Stats.IntelRdtStats will be assigned to nil when intelRdtManager.GetStats() returns nil, nil.
And in events command, the caller convertLibcontainerStats() will handle the case when libcontainer.Stats.IntelRdtStats = nil